index.js

```diff
@@ -214,6 +239,54 @@ const lookUpCommand = (message) => {
     });
 };
 
+/*
+
+  const topFiveCommand = (message) => {};
+    The user uses this command to select up to 5 cards  from their inventory to display
+    any user can use the lookup command to look at another users top 5 cards, but only the owner of cards can select which cards go into their top 5
+    
+    To input a card to your top 5, the command must be entered as !showcase blue-eyes white dragon 
+
+
+
+*/
+
+const removeCardCommand = (message) => {
+  const removeThisCard = message.content.split(" ").slice(1).join(" ");
```

Comment:
nit: hard to describe why naming conventions are what they are, but would normally be more like `cardToRemove` or `cardName` (since we understand from the context of this function that its likely the card being removed, but what we may not know is whether its the name string, or the card object)